### DISCUSSION TITLE
keyball61でトラックボール周りの特殊なキーコードとRemap上の表記が1つずつズレている

### DISCUSSION THREAD

**Discussion by deg84** [Yowkees/keyball#242] [Q&A]
全てを確認したわけではありませんが、恐らく1つずつコードがズレているようなんですが、解消する方法ありますか？

見ているドキュメントはこれです。

https://github.com/Yowkees/keyball/blob/fd717ea600239a08642e2cbd2df01f9d69bf236b/keyball61/doc/rev1/buildguide_jp.md#%EF%BC%91%EF%BC%91keyball%E3%81%AE%E4%BD%BF%E3%81%84%E6%96%B9

`SCRL_TO`を設定したいんですが、`0x5DAB`ではダメで、`0x5DAC`を設定する必要がありました。
`0x5DAB`は`CPI_D1K`と認識されているようでした。

試したファームウェア
1. https://remap-keys.app/catalog/RZSU1CrvEW4lns0ww5BM/firmware の`SHA256: e08e4e3ce365d343ac9f065956fa62f0be0446229316fcc1bce0a2b480d32b29`のもの
2. https://github.com/Yowkees/keyball/releases/download/keyball61%2Fv1.1.1/keyball_keyball61_via.hex

キーマップの設定にはRemapを使っています。
初回のファームウェア書き込みにはQMK Toolboxを使用してます。Mac版です。

現象としてはこちらのディスカッション内容と同じように思います。
https://github.com/Yowkees/keyball/discussions/129
こちら解決していない＋Keyball39のことのようなので新たにディスカッション開始させて頂きました。

解決策はありますか？
ドキュメントが間違っている可能性はあるんでしょうか？

よろしくお願いします。

**Comment by Yowkees** [comment]
キーコードのズレについては報告をいただいておりまして、コンパイルするQMKバージョンに依存します。

最新QMKで修正したいのですが、Remapが最新QMKに対応していないため、
その他の不具合が出そうなので修正を見送っております。

特殊キーのみの不具合ですので、お手数ですが１つズレていることを念頭にキーマップをご検討いただければ幸いです。

以上、お手数ですが、ご対応お願いいたします。

**Comment by Hideck-Github** [comment]
Hideck と申します

私も同じ現象にぶつかりました。Try&Errorで解決しました。
同じＱ＆Ａが増えるでしょうら、注意書きをしておいていただくとお互いに無駄な時間が減っていいと思います

ご検討のほどよろしくお願いします。
